services/ingestion/engine_test.go

```diff
@@ -3,13 +3,14 @@ package ingestion
 import (
```

Comment:
can you add a test with the spork clients?

Reply:
I was thinking how to do that, and I do plan to, but it requires mocking out the spork client which now is not defined as an interface. But nonetheless I will add to this PR (if it was urgent I planned to test locally and manually, but since now we got EVM GW up and running with snapshot I will take the time to test).